rclcpp/src/rclcpp/time_source.cpp

```diff
@@ -37,6 +37,7 @@ TimeSource::TimeSource(std::shared_ptr<rclcpp::Node> node)
 : ros_time_active_(false)
 {
   this->attachNode(node);
+  clock_subscription_.reset();
```

Comment:
Is there a reason this has to go after `this->attachNode(node);`? It would be better (in my opinion) to do this in the member initialization section of the constructor as `clock_subscription_(nullptr)`.

Reply:
nah, there wasn't a reason: 73d4a7bf0c945a03a1a35d189394e2a9f5299bb0